### PR TITLE
Create ProblemList.openjdk18.txt

### PR DESCRIPTION
**Pull request by SehrishHussain**
I drilled down to individual failed test cases in JUnit test results and prepared the "ProblemList.openjdk18.txt" file. There were some test cases that were already included in ProblemList.openjdk17 file that I didn't add. 
Note: This file is a work in progress.
Addressing issue: #3019

### CODE REVIEW

**Comment by SehrishHussain** [comment]
@smlambert not sure if this file can be merged at this point.

File: openjdk/excludes/ProblemList_openjdk18.txt [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+
+
+#java/nio/channels/DatagramChannel/MulticastSendReceiveTests.java is excluded for two different reasons on different platforms https://github.com/adoptium/aqa-tests/issues/1016
+#java/nio/channels/DatagramChannel/MulticastSendReceiveTests.java https://github.com/adoptium/infrastructure/issues/699 linux-s390x
+java/nio/channels/DatagramChannel/MulticastSendReceiveTests.java https://github.com/adoptium/aqa-tests/issues/1267 macosx-all,linux-s390x
+#java/nio/channels/DatagramChannel/Promiscuous.java is excluded on macosx due to different issue	https://github.com/adoptium/aqa-tests/issues/602
+java/nio/channels/DatagramChannel/Promiscuous.java              https://github.com/adoptium/infrastructure/issues/699    linux-s390x,macosx-all	
+java/nio/file/spi/SetDefaultProvider.java	https://github.com/adoptium/aqa-tests/issues/2871 linux-arm
+java/net/ipv6tests/UdpTest.java https://bugs.openjdk.java.net/browse/JDK-8198266 linux-s390x
+******
```
Comment (andrew-m-leonard) [inline openjdk/excludes/ProblemList_openjdk18.txt:182]: I'm not sure this line with ******'s is going to be processed ok, presume it's meant to be ####'s ?

**Review by andrew-m-leonard — CHANGES_REQUESTED** [review]

File: openjdk/excludes/ProblemList_openjdk18.txt (hunk 1 above)
Comment (SehrishHussain) [reply openjdk/excludes/ProblemList_openjdk18.txt:182]: @andrew-m-leonard yes I missed this one! TY

File: openjdk/excludes/ProblemList_openjdk18.txt [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+compiler/graalunit/ReplacementsTest.java https://github.com/adoptium/aqa-tests/issues/2804 generic-all
+compiler/graalunit/UtilTest.java https://github.com/adoptium/aqa-tests/issues/2804 generic-all
+compiler/unsafe/UnsafeCopyMemory.java   https://github.com/adoptium/aqa-tests/issues/2804   aix-ppc64,linux_s390x
+
+compiler/c2/cr6340864/TestIntVectRotate.java    https://github.com/adoptium/aqa-tests/issues/3004 windows_x86-32
+compiler/c2/cr6340864/TestLongVectRotate.java   https://github.com/adoptium/aqa-tests/issues/3004 windows_x86-32
+compiler/c2/irTests/TestPostParseCallDevirtualization.java  https://github.com/adoptium/aqa-tests/issues/3004 windows_x86-32
+compiler/loopstripmining/TestPinnedUseInOuterLSMUnusedBySfpt.java   https://github.com/adoptium/aqa-tests/issues/3004 windows_x86-32
+compiler/tiered/TieredLevelsTest.java   https://github.com/adoptium/aqa-tests/issues/3004 windows_x86-32
+compiler/vectorapi/TestNoInline.java    https://github.com/adoptium/aqa-tests/issues/3004 windows_x86-32
```
Comment (sophia-guo) [inline openjdk/excludes/ProblemList_openjdk18.txt:379]: I believe windows_x86 is for windows32.

**Comment by sophia-guo** [comment]
rerun : https://ci.adoptopenjdk.net/view/Test_openjdk/job/Test_openjdk18_hs_sanity.openjdk_x86-64_linux/27/consoleFull . Failure is about the jtreg version, which is not related with this PR. Pretty sure we will continue to update this file ProblemList_openjdk18.txt. For now LGTM.

File: openjdk/excludes/ProblemList_openjdk18.txt (hunk 2 above)
Comment (sophia-guo) [reply openjdk/excludes/ProblemList_openjdk18.txt:379]: So could you update windows_x86-32 to windows_x86?

**Review by sophia-guo — APPROVED** [review]

**Review by andrew-m-leonard — APPROVED** [review]
lgtm

**Pull request merged**